src/scitokens.cpp

```diff
@@ -207,34 +276,347 @@ class XrdAccSciTokens : public XrdAccAuthorize
                       const XrdSecEntity    *Entity,
                       const char            *path,
                       const Access_Operation oper,
-                            XrdOucEnv       *Env=0)
+                            XrdOucEnv       *Env=0) override
     {
         return 0;
     }
 
     virtual int         Test(const XrdAccPrivs priv,
-                             const Access_Operation oper)
+                             const Access_Operation oper) override
     {
         return 0;
     }
 
 private:
 
+    bool GenerateAcls(const std::string &authz, uint64_t &cache_expiry, AccessRulesRaw &rules, std::string &username) {
+        if (strncmp(authz.c_str(), "Bearer%20", 9)) {
+            return false;
+        }
+
+        // Does this look like a JWT?  If not, bail out early and
+        // do not pollute the log.
+        bool looks_good = true;
+        int separator_count = 0;
+        for (auto cur_char = authz.c_str() + 7; *cur_char; cur_char++) {
```

Comment:
Why 7 and not 9?  Isn't 9 the start of the bearer token (you used 9 above when testing for `Bearer%20`)

Reply:
You're correct, the offset is 9.